lib/hammer_cli_katello/host_extensions.rb

```diff
@@ -28,6 +28,15 @@ module HostExtensions
         from :content_facet_attributes do
           field :content_view_name, _('Content View')
           field :lifecycle_environment_name, _('Lifecycle Environment')
+          from :errata_counts do
```

Comment:
The fields can be in single block like:
```ruby
          from :errata_counts do
            field :security, _("Security"), nil, :sets => ['ALL']
            field :bugfix, _("Bugfix"), nil, :sets => ['ALL']
            field :enhancement, _("Enhancement"), nil, :sets => ['ALL']
          end
```
Was there any reason to have it separate?

Reply:
Nops, just testing and improving my RoR skill. Thanks for the heads up and advice.

Reply:
about the new feature, I believe the best place will be via our official docs. I can fill the BZ requesting this info once the feature being available.